### ISSUE TITLE
Error during parsing

### ISSUE THREAD

**Issue by t56k** [inv2004/ttop#23]
I get this response when running `ttop`:

```
error during parsing /proc/diskstats: procfs.nim(365, 16) `scanf(line, "$s$i $s$i $w $i $i $i $i $i $i $i $i $i $i", tmp, tmp, name, tmp,
      tmp, tmp, read, tmp, tmp, tmp, write, tmp, total)`
procfs.nim(124)          diskInfo
```

This is on Arch 6.2.7. Let me know if you need more info or if I can help.

**Comment by inv2004** [comment]
@t56k 
Thank you for the bug report.

Could you please attach/show your diskstats if it does not contain any secret information.

Regards,

**Comment by t56k** [comment]
@inv2004: you bet:

```
❯ cat /proc/diskstats
   8       0 sda 169644 1294 20262360 1052919 245000 239 14522640 125523 0 836527 1702415 0 0 0 0 6990 523971
   8       1 sda1 169471 1294 20256976 1052700 245000 239 14522640 125523 0 836360 1178224 0 0 0 0 0 0
   8       9 sda9 58 0 712 103 0 0 0 0 0 124 103 0 0 0 0 0 0
 259       0 nvme0n1 42209 1623 3455133 15835 617519 6161 27609027 838441 0 537687 954967 6 0 912296 6 65235 100684
 259       1 nvme0n1p1 249 998 16055 43 13 0 11 2 0 70 52 6 0 912296 6 0 0
 259       2 nvme0n1p2 41863 625 3435326 15782 617506 6161 27609016 838438 0 537634 854221 0 0 0 0 0 0
 259       3 nvme1n1 6159 177 2563672 4292 334782 22619 6857448 537742 0 555427 714693 46625 0 874374184 55469 65233 117188
 254       0 dm-0 42134 0 3434048 18478 621996 0 27609016 1562983 0 538940 1581461 0 0 0 0 0 0
 253       0 zram0 294 0 2352 0 1 0 8 0 0 10 0 0 0 0 0 0 0
```

**Comment by inv2004** [comment]
@t56k Should be fixed here: fbe1fbe009cba2e87c13f2cca56f845d47c253dc

New release is v1.1.2, please update your version

Thank you,

**Comment by t56k** [comment]
Works great, thanks so much!
